client/src/components/commonComponents/navBar/style.js

```diff
@@ -0,0 +1,60 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  header: {
+    backgroundColor: '#000000',
+    color: 'white',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  logo: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginTop: '20px',
+    fontFamily: 'Quicksand',
+    fontSize: '25px',
+  },
+  img: {
+    width: '60px',
+    height: '60px',
+  },
+
+  list: {
+    fontSize: '12px',
+    [theme.breakpoints.up(600 + theme.spacing(2) * 2)]: {
+      marginLeft: '65px',
+    },
+  },
+  item: {
+    color: '#90B27B',
+    fontWeight: 'bold',
+  },
+  intruduction: {
+    fontFamily: 'Quicksand',
```

Comment:
For the font family, we're using **_Proxima Nova_**, pls import it instead of Quicksand  
_'check  the style guide in Figma'_